design-tokens/tokens/layer3/dark.json

```diff
@@ -46,7 +46,7 @@
         }
       }
     },
-    "modal": {
```

Comment:
I think this should shift to layer 2 if it's not specific to a component anymore